test/extended/cluster/config/master-vert.yaml

```diff
@@ -0,0 +1,60 @@
+provider: local
```

Comment:
Put these in test/extended/testdata/cluster/.  When you do that my PR to bindata these will automatically pick them up.  Use FixturePath() to access the files.